### ISSUE TITLE
Threads parked, seem to block forever

### ISSUE THREAD

**Issue by afandian** [http-kit/http-kit#347]
I have a few threads (around 5) in which I'm doing

    my_project:35    (let [result @(org.httpkit.client/get "http://example.com")]

I'm using threads rather than promises in `go` blocks because an external library (Apache Kafka Client) needing to be run in a few threads in parallel.

After some minutes of processing inputs and making HTTP calls, the whole thing grinds to a halt. My threads all look like this:

    sun.misc.Unsafe.park(Native Method)
    java.util.concurrent.locks.LockSupport.park(LockSupport.java:175)
    java.util.concurrent.locks.AbstractQueuedSynchronizer.parkAndCheckInterrupt(AbstractQueuedSynchronizer.java:836)
    java.util.concurrent.locks.AbstractQueuedSynchronizer.doAcquireSharedInterruptibly(AbstractQueuedSynchronizer.java:997)
    java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1304)
    java.util.concurrent.CountDownLatch.await(CountDownLatch.java:231)
    clojure.core$promise$reify__7005.deref(core.clj:6823)
    clojure.core$deref.invokeStatic(core.clj:2228)
    clojure.core$deref.invoke(core.clj:2214)
    my_project.web$fetch.invokeStatic(web.clj:35)

There's another thread stuck here:

    sun.security.ssl.SSLEngineImpl.readNetRecord(SSLEngineImpl.java:850)
    sun.security.ssl.SSLEngineImpl.unwrap(SSLEngineImpl.java:781)
    javax.net.ssl.SSLEngine.unwrap(SSLEngine.java:624)
    org.httpkit.client.HttpsRequest.unwrapRead(HttpsRequest.java:35)
    org.httpkit.client.HttpClient.doRead(HttpClient.java:131)
    org.httpkit.client.HttpClient.run(HttpClient.java:377)
    java.lang.Thread.run(Thread.java:748)

I don't know enough about the internals of HTTP Kit to guess the connection, it looks like all of the promises are blocking on the above code, which has stalled for some reason. 

I think my use of HTTP Kit is normal enough, so I think it's reasonable to expect this not to happen?

(I've also asked over at StackOverflow https://stackoverflow.com/questions/45619098/threads-parked-with-http-kit but I think this could be a bug.)

**Comment by ptaoussanis** [comment]
Hi there,

Sorry, there's not enough to go on here to properly help debug. Would suggest providing a reproducible example if possible?

**Comment by ptaoussanis** [comment]
Closing as part of issue triage due to inactivity
